src/Analyzers/CSharp/CodeFixes/UseImplicitOrExplicitType/UseExplicitTypeCodeFixProvider.cs

```diff
@@ -205,7 +205,7 @@ private static ExpressionSyntax GenerateTupleDeclaration(ITypeSymbol typeSymbol,
 
         return TupleExpression(
             OpenParenToken.WithTrailingTrivia(),
-            SeparatedList(builder.ToImmutable(), separatorBuilder.ToImmutableAndFree()),
+            SeparatedList(builder, separatorBuilder),
```

Comment:
this guy doesn't need to realize the immutable arrays.  so this can stay nicely as ArrayBuilders.